packages/phasor/src/surface.ts

```diff
@@ -202,7 +202,7 @@ export class SurfaceManager {
 
   updateElement<T extends keyof IPhasorElementType>(
```

Comment:
Can we always call `doc.transact` in `updateElement`?

Reply:
https://github.com/toeverything/blocksuite/pull/2677#issuecomment-1557612615